docs/usage/tool_params.md

```diff
@@ -52,7 +52,16 @@ Seqtk is used for both the sub-sampling of reads and conversion of fasta files t
     - seed: A seed value for sub-sampling
     - reads_ext: Extension of reads after sub-sampling, do not touch alter this unless doing pipeline development.
     - assembly_fastq: Extension of the fastas after being converted to fastq files. Do no touch this unless doing pipeline development.
-    - report_tag: Name of seqtk data in the final summary report. Do no touch this unless doing pipeline development.
+    - report_tag: Name of seqtk data in the fi nal summary report. Do no touch this unless doing pipeline development.
+
+### Rasusa
+For long read data Rasusa is used for down sampling as it take read length into consideration when down sampling.
+
+- rasusa
+    - singularity: singularity container for rasusa
+    - docker: docker container for rasusa
+    - seed: A seed value for sub-sampling
+    - reads_ext: The extension of the generated fastq files. Do no touch this unless doing pipeline development.
```

Comment:
downsamping is probably one word
Singularity, Docker capitalization
Also consider: "do not change this" instead of "do not touch this"

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/125/commits/667b67d77d74bffb3d9ccedde33cd0f95ee0a188